glad/documentation.py

```diff
@@ -0,0 +1,171 @@
+from glad.parse import SpecificationDocs, CommandDocs, xml_parse
+from glad.util import prefix, memoize, raw_text
+from shutil import rmtree
+import glad.util
+import subprocess
+import re
+
+
+class DocsGL(SpecificationDocs):
+    URL = 'https://github.com/BSVino/docs.gl.git'
+    CACHED = True # Only clones the DocsGL repo once
+    API = 'gl'
+    docs = dict()
```

Comment:
I think you wanted this on the instance not class?

Reply:
Removed, now I use a `DocumentationSet` similar to `FeatureSet`